stats/stats.py

```diff
@@ -271,16 +278,20 @@ def documents_paid_date(self, queryset):
             else:
                 granulation_value = None
 
-            self.get_stats_data(granulation_arguments, customer_name, document.currency, None,
-                                granulation_value, document.id, doc, None)
+            additional_field = {"payment_day": document.paid_date.strftime('%d')}
+
+            self.get_granulation_tuple(granulation_arguments, customer_name, document.currency,
```

Comment:
Because there are many arguments in this call, it would be nice to specify their name, in other words send them as keyword args.